bundle/regal/config/exclusion_test.rego

```diff
@@ -91,21 +91,6 @@ test_excluded_file_cli_overrides_config if {
 		with data.eval.params as object.union(params, {"ignore_files": [""]})
 }
 
-test_excluded_file_using_uri if {
-	conf := {"rules": {"test": {"rule": {
-		"level": "error",
-		"ignore": {"files": ["foo/**/p.rego"]},
-	}}}}
-
-	config.excluded_file("test", "rule", "file:///workspace/folder/foo/bar/p.rego") with config.merged_config as conf
-}
-
-test_not_excluded_file_using_uri if {
-	conf := {"rules": {"test": {"rule": {"level": "error"}}}}
-
-	not config.excluded_file("test", "rule", "file:///workspace/folder/foo/bar/p.rego") with config.merged_config as conf
-}
```

Comment:
I did not expect these tests to have to be removed, but perhaps moved to the tests of main. Wr could certainly use some coverage here.

Reply:
I think that 72cc35b7af3b995c09e0f08a5bd432f9aca302d2 is a good improvement here. I've brought back some tests, though they are now in main.rego as config is not meant to know about URIs any more.

I have also renamed the rootDir refs to pathPrefix to make it clearer that we're supporting URI or path prefixes with these.